TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -1050,46 +1060,29 @@ public override Token GetTokenStartingFrom(int startIndex)
                         return new Token(TokenType.MultiplyOperator, startIndex, startIndex, tokensLine);
                     }
                 case '%':
-                    if (line.Length >= currentIndex + 2 && line[currentIndex + 1] == '>' && line[currentIndex + 2] == '>')
```

Comment:
This code is not useful anymore ?

Reply:
Yes and bugged.
The diagnostics for `WrongFormalizedCommentMarckupPosition` and `WrongMultilineCommentMarckupPosition` are created elsewhere:

https://github.com/TypeCobolTeam/TypeCobol/blob/ee6b01617bc4155372a9429b7c786729e4f7ebbd/TypeCobol/Compiler/Scanner/Scanner.cs#L874-L883

https://github.com/TypeCobolTeam/TypeCobol/blob/ee6b01617bc4155372a9429b7c786729e4f7ebbd/TypeCobol/Compiler/Scanner/Scanner.cs#L916-L925

And it does not concern starting tags but closing ones (and the check on index value is wrong too).